mitodl/open-discussions
# Implement uploading channel avatar and banner

#### What are the relevant tickets?
Part of #927 

#### What's this PR do?
Provides the UI to edit channel avatars and banners

#### How should this be manually tested?
Edit the channel avatar and banner. Note that if you do not click 'Save' the changes should be discarded. (There isn't clear indication of this at the moment.)
